### PR TITLE
Add strategy_id to terraformer

### PR DESCRIPTION
**Pull request by Lesigh-3100**
The PR that this branch is based on introduced the "strategy_id" field into the static data (and Pool class) data structure. This field is only applicable for Carbon pools, and is set to 0 for all other pools to maintain consistent typing (int). This change to the Terraformer adds the missing field when pool data is generated.

I have added a corresponding test that ensures that this field is set for the four main data processing functions in the Terraformer. These tests can be expanded further into a test suite for the Terraformer, however, that is out of the scope of this PR.

### CODE REVIEW

**Review by barakman — CHANGES_REQUESTED** [review]
Please explain what this change addresses and/or fix.
Please add a corresponding test of some sort.

**Comment by Lesigh-3100** [comment]
> Please explain what this change addresses and/or fix. Please add a corresponding test of some sort.
I've updated the explanation of this PR & added tests.

**Review by barakman — CHANGES_REQUESTED** [review]
Tests are failing with:
```
TypeError: get_token_and_fee() got an unexpected keyword argument 'exchange_name'
```
Which is pretty weird, since this function does take an input argument by that name.
But it nevertheless requires looking into it.

File: fastlane_bot/tests/test_072_TestTerraformer.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+'''
+import pytest
+import asyncio
+
+from unittest.mock import MagicMock, AsyncMock
+
+from json import dumps
+from pytest import fixture
+from dataclasses import dataclass
+from run_blockchain_terraformer import organize_pool_details_balancer, organize_pool_details_uni_v2, organize_pool_details_solidly_v2, organize_pool_details_uni_v3
```
Comment (barakman) [inline fastlane_bot/tests/test_072_TestTerraformer.py:13]: The following modules aren't being used; please remove the corresponding `import` statements:
```
import pytest
import asyncio
from json import dumps
from pytest import fixture
from dataclasses import dataclass
```

**Review by barakman — APPROVED** [review]

File: fastlane_bot/tests/test_072_TestTerraformer.py (hunk 1 above)
Comment (Lesigh-3100) [reply fastlane_bot/tests/test_072_TestTerraformer.py:7]: Done - thank you.

**Pull request merged**